tests/functionaltests/MainViewController.mm

```diff
@@ -27,9 +27,9 @@ @interface MainViewController ()
 @implementation MainViewController
 
 - (void)viewDidLoad {
-    self.view.backgroundColor = [UIColor lightGrayColor];
-
     [super viewDidLoad];
+
+    self.view.backgroundColor = [UIColor purpleColor];
```

Comment:
>purpleColor [](start = 41, length = 11)

I could revert this but the upside is that it draws your attention when dismiss the UIViewController ;-)